python/cog/predictor.py

```diff
@@ -23,28 +23,24 @@
 
 import structlog
 
-import cog.code_xforms as code_xforms
-
 try:
     from typing import get_args, get_origin
 except ImportError:  # Python < 3.8
     from typing_compat import get_args, get_origin  # type: ignore
 
+import pydantic
 import yaml
-from pydantic import BaseModel, Field, create_model
+from pydantic import BaseModel, ConfigDict, Field, create_model
```

Comment:
```suggestion
from pydantic import BaseModel, Field, create_model
```

Move `ConfigDict` import statement under this conditional